### PR TITLE
chore: update undici to v6.14.1

### PR DESCRIPTION
**Pull request by benoitf**
### What does this PR do?
update undici to v6.14.1

### How to test this PR?

- [ ] Tests are covering the bug fix or the new feature

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -2114,7 +2114,7 @@
     "@docusaurus/theme-search-algolia" "3.2.1"
     "@docusaurus/types" "3.2.1"
 
-"@docusaurus/react-loadable@5.5.2":
+"@docusaurus/react-loadable@5.5.2", "react-loadable@npm:@docusaurus/react-loadable@5.5.2":
```
Comment (jeffmaury) [inline yarn.lock]: Seems unrelated to me

Comment (benoitf) [reply yarn.lock]: so anyway this change has already be merged (if I rebase it should disappear)

Comment (benoitf) [reply yarn.lock]: https://github.com/containers/podman-desktop/commit/d95a0e588f6b21620f5741712e2d2dfd733fed9c

**Comment by benoitf** [comment]
rebased to avoid the change @jeffmaury

**Review by jeffmaury — APPROVED** [review]
LGTM

**Review by deboer-tim — APPROVED** [review]
